### PR TITLE
Fix docs and typos for `PymunkPhysicsEngine.add_sprite` and  `PymunkPhysicsEngine.add_sprite_list` respectively

### PR DESCRIPTION
**Pull request by Ibrahim2750mi**
Relevant conversations:

+ https://discord.com/channels/458662222697070613/705076586986078350/1086340733251878973
+ https://discord.com/channels/458662222697070613/458662458198982676/1086622783288651776

### CODE REVIEW

File: arcade/pymunk_physics_engine.py [hunk 1]
```diff
@@ -70,21 +70,21 @@ def add_sprite(self,
...
+            :param damping: See class docs.
+            :param gravity: See class docs.
             :param max_velocity: The maximum velocity of the object.
-            :param max_horizontal_velocity: maximum velocity on the x axis
-            :param max_vertical_velocity: maximum velocity on the y axis
-            :param radius:
-            :param collision_type:
+            :param max_horizontal_velocity: Maximum velocity on the x axis.
+            :param max_vertical_velocity: Maximum velocity on the y axis.
+            :param radius: Radius for the shape created for the sprite.
```
Comment (pushfoo) [inline arcade/pymunk_physics_engine.py]: What's the unit on this? Pixels?

Comment (Ibrahim2750mi) [reply arcade/pymunk_physics_engine.py]: It should be, doesn't makes sense to use any other unit.

Comment (pushfoo) [reply arcade/pymunk_physics_engine.py]: I'd add that to the docstring.

Comment (Ibrahim2750mi) [reply arcade/pymunk_physics_engine.py]: I mean if you say so, I will add it. There is no harm in explicitly clearing something. Also if you had the question others will too, so I will add it.

Comment (Ibrahim2750mi) [reply arcade/pymunk_physics_engine.py]: Updated in https://github.com/pythonarcade/arcade/pull/1642/commits/fed3d1dece977cd4aaa544db4797a5ec62feb310

**Review by pushfoo — APPROVED** [review]
:+1:

**Review by pushfoo — APPROVED** [review]

**Pull request merged**